discovery/spec.md

```diff
@@ -588,10 +588,17 @@ The result MUST be a JSON object of the following form:
 ```
 {
   "servicefilterattributes": [ "name", ... ],
-  "pagination": "true"
+  "pagination": true
 }
 ```
 
+The following additional constraints apply:
+- The list of feature names are case sensitive.
+- The `servicefilterattribute` MUST be present, and MUST have at least the
+  `name` attribute in its list. The list of attribute names are case sensitive.
```

Comment:
Remove the second sentence here, given that it's already above?

Reply:
I think this is different. This is talking about filter attribute names, but the previous one is talking about feature names. But I did reword the sentence per your previous comment.